datadog_checks_dev/datadog_checks/dev/tooling/files/manifest.py

```diff
@@ -0,0 +1,78 @@
+# (C) Datadog, Inc. 2018
+# All rights reserved
+# Licensed under a 3-clause BSD style license (see LICENSE)
+import os
+import uuid
+
+from .utils import File
+
+TEMPLATE_IN = """\
```

Comment:
can we put this (and the others) in a separate file and read it in? I think it'll be easier to upkeep than as a string